packages/web/src/store/selectors/websiteResults.ts

```diff
@@ -52,7 +53,8 @@ const makeSelectScanPackagesByUrl = () =>
           !pkg.versionSet.some(
             (ver) => pkg.registryMetadata && semver.eq(pkg.registryMetadata.latestVersion, ver)
           ),
-        vulnerable: (scanData.vulnerabilities[pkg.name]?.length ?? 0) > 0,
+        vulnerable: (scanData.vulnerabilities[pkg.name]?.length ?? 0) > 0, // TODO: Drop
+        vulnerabilities: scanData.vulnerabilities[pkg.name] ?? [],
```

Comment:
I guess that may conflict with #80

Reply:
Updated to use this field across the component.